src/applications/disability-benefits/all-claims/pages/uploadPersonalPtsdDocuments.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import fileUploadUI from 'us-forms-system/lib/js/definitions/file';
+import environment from '../../../../platform/utilities/environment';
+
+import { DocumentDescription } from '../content/uploadPtsdDocuments';
+import { PtsdNameTitle } from '../content/ptsdClassification';
+
+const FIFTY_MB = 52428800;
+
+export const uiSchema = {
+  'ui:title': ({ formData }) => (
+    <PtsdNameTitle formData={formData} formType="781a" />
+  ),
+  'ui:description': DocumentDescription,
+  ptsd781a: fileUploadUI('', {
+    itemDescription: 'PTSD 781a form',
+    hideLabelText: true,
+    fileUploadUrl: `${environment.API_URL}/v0/upload_supporting_evidence`,
+    fileTypes: [
+      'pdf',
+      'jpg',
+      'jpeg',
+      'png',
+      'gif',
+      'bmp',
+      'tif',
+      'tiff',
+      'txt',
+    ],
+    maxSize: FIFTY_MB,
+    createPayload: file => {
+      const payload = new FormData();
+      payload.append('supporting_evidence_attachment[file_data]', file);
+
+      return payload;
+    },
+    parseResponse: (response, file) => ({
+      name: file.name,
+      confirmationCode: response.data.attributes.guid,
+    }),
+    // this is the uiSchema passed to FileField for the attachmentId schema
+    // FileField requires this name be used
+    attachmentSchema: {
+      'ui:title': 'Document type',
+    },
+    // this is the uiSchema passed to FileField for the name schema
+    // FileField requires this name be used
+    attachmentName: {
+      'ui:title': 'Document name',
+    },
+  }),
+};
+
+export const schema = {
+  type: 'object',
+  required: ['ptsd781a'],
+  properties: {
+    ptsd781a: {
```

Comment:
would be good to get this and any other non-`view:` schema into https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/all-claims/config/schema.js to make the transition to `vets-json-schema` easier once we decide to put the schema in there.